### PR TITLE
feat(peer-managers): move group members actions to peer managers

### PR DESCRIPTION
**Pull request by rockjam**

### CODE REVIEW

File: actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/GroupPeerManager.scala [hunk 1]
```diff
@@ -24,10 +28,16 @@ case class GroupPeerManagerRegion(ref: ActorRef)
 
 object GroupPeerManager {
 
+  type SequenceStateDate = (SequenceState, Long)
```
Comment (prettynatty) [inline actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/GroupPeerManager.scala]: Declare it in SeqUpdatesManager

File: actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala [hunk 2]
```diff
@@ -46,10 +47,26 @@ object PeerManager {
...
-    @(Tag @field)(0) group:            models.Group,
-    @(Tag @field)(1) joiningUserId:    Int,
-    @(Tag @field)(2) joiningUserAuthId:Long,
-    @(Tag @field)(3) invitingUserId:   Int
+    @(Tag @field)(0) joiningUserId:    Int,
+    @(Tag @field)(1) joiningUserAuthId:Long,
+    @(Tag @field)(2) invitingUserId:   Int
+  ) extends Message
+
+  private[peermanagers] case class InviteToGroup(
```
Comment (prettynatty) [inline actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala]: need kryo mappings

File: actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala [hunk 3]
```diff
@@ -46,10 +47,26 @@ object PeerManager {
...
+    @(Tag @field)(0) group:        models.FullGroup,
+    @(Tag @field)(1) inviteeUserId:Int,
+    @(Tag @field)(2) client:       AuthorizedClientData,
+    @(Tag @field)(3) randomId:     Long
+  ) extends Message
+  private[peermanagers] case class KickUser(
```
Comment (prettynatty) [inline actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala]: need kryo mappings

File: actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala [hunk 4]
```diff
@@ -46,10 +47,26 @@ object PeerManager {
...
+    @(Tag @field)(0) kickedUserId:Int,
+    @(Tag @field)(1) client:      AuthorizedClientData,
+    @(Tag @field)(2) randomId:    Long
+  ) extends Message
+
+  private[peermanagers] case class LeaveGroup(
```
Comment (prettynatty) [inline actor-server/actor-peer-managers/src/main/scala/im/actor/server/peermanagers/PeerManager.scala]: need kryo mappings

**Comment by rockjam** [comment]
@prettynatty updated

**Comment by rockjam** [comment]
@prettynatty updated

**Pull request merged**